content/docs/esc/_index.md

```diff
@@ -15,7 +15,7 @@ aliases:
 
 Pulumi ESC (Environments, Secrets, and Configuration) allows teams to tackle secrets and configuration complexity for modern cloud environments, alleviating maintenance burden and reducing costly mistakes, and creating a "secure by default" posture. Pulumi ESC is a new category of configuration as code product, motivated by our experience working with hundreds of Pulumi IaC customers to address their needs in managing secrets and configuration at scale within their Pulumi infrastructure and across other cloud applications and infrastructure projects.
```

Comment:
should the URL instead be "secrets-management"?

Reply:
This is the root docs URL path for ESC I would not suggest we change this, however I would suggest we create a "secrets-management" subpage in concepts to help with SEO.

Reply:
Issue filed here: https://github.com/pulumi/docs/issues/11524